### PR TITLE
v8o: Add a testcase to use auxilary image in MII domain

### PR DESCRIPTION
**Pull request by sankarpn**
This PR adds a new test in existing VZ IT class integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java  

https://build.weblogick8s.org:8443/view/WKO(Nightly)/job/v8o-kind-nightly/95
https://build.weblogick8s.org:8443/view/WKO(Nightly)/job/v8o-kind-nightly/104/

### CODE REVIEW

File: integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java [hunk 1]
```diff
@@ -197,7 +217,202 @@ void testCreateVzMiiDomain() {
...
     String appUrl = "https://" + host + "/sample-war/index.jsp --resolve " + host + ":443:" + address;
     assertTrue(verifyVzApplicationAccess(appUrl, message), "Failed to get access to sample application");
+
+  }
+
+  /**
+   * Create a WebLogic domain VerrazzanoWebLogicWorkload component in verrazzano with auxilary images.
+   */
+  @Test
+  @DisplayName("Verrazzano mill domain with auxilary image")
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java:227]: typo here:
mill -> mii

File: integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java [hunk 2]
```diff
@@ -197,7 +217,202 @@ void testCreateVzMiiDomain() {
...
+  void testVzMiiAuxilaryImage() {
+    String auxDomainUid = "aux-domain";
+    String miiAuxiliaryImage1Tag = "image1" + MII_BASIC_IMAGE_TAG;
+    String miiAuxiliaryImage1 = MII_AUXILIARY_IMAGE_NAME + ":" + miiAuxiliaryImage1Tag;
+    String miiAuxiliaryImage2Tag = "image2" + MII_BASIC_IMAGE_TAG;
+    String miiAuxiliaryImage2 = MII_AUXILIARY_IMAGE_NAME + ":" + miiAuxiliaryImage2Tag;
     
+    final int replicaCount = 2;
+    final String adminServerPodName = auxDomainUid + "-admin-server";
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java:236]: You can use ADMIN_SERVER_NAME_BASE here.

File: integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java [hunk 3]
```diff
@@ -197,7 +217,202 @@ void testCreateVzMiiDomain() {
...
+    final String managedServerPrefix = auxDomainUid + "-managed-server";
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java:237]: You can use MANAGED_SERVER_NAME_BASE here.

File: integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java [hunk 4]
```diff
@@ -197,7 +217,202 @@ void testCreateVzMiiDomain() {
...
+    createAndPushAuxiliaryImage(MII_AUXILIARY_IMAGE_NAME, miiAuxiliaryImage2Tag, witParams);
+
+    // admin/managed server name here should match with model yaml
+    String auxiliaryImagePath = "/auxiliary";
+    String clusterName = "cluster-1";
+
+    // create domain custom resource using 2 auxiliary images
+    logger.info("Creating domain custom resource with domainUid {0} and auxiliary images {1} {2}",
+        auxDomainUid, miiAuxiliaryImage1, miiAuxiliaryImage2);
+    DomainResource domainCR = CommonMiiTestUtils.createDomainResourceWithAuxiliaryImage(
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java:284]: You can use static import here:
CommonMiiTestUtils.createDomainResourceWithAuxiliaryImage

File: integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java (hunk 1 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java]: fixed

File: integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java (hunk 2 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java]: fixed

File: integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java (hunk 4 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java]: fixed

**Review by anpanigr — APPROVED** [review]

**Pull request merged**